### DISCUSSION TITLE
[Feature Request] Kanban board in project view

### DISCUSSION THREAD

**Discussion by OdinJorna** [NicolasPetton/Projectify#9] [Ideas]
Currently, the project view lists all the todos that are sorted into that project. With bigger projects, it can get tricky to track all the todos. Or to keep an overview when the list gets long. Allowing users to sort their todos on a board would help bring more organisation to their projects. Todos can be tagged, which allows them to be sorted into different columns, so when users switch been list- and board mode they retain in the correct column. A drag and drop mechanic would be very useful here.

Do you think something like this would be achievable? Being able to sort todos that don't have a project yet by dragging and dropping into the project card on the dashboard feels like a very similar mechanic already.

**Comment by kookma** [comment]
There are two version of Kanban in Tiddlywiki.

- https://ibnishak.github.io/Tesseract/projects/tekan/Tekan.html  (popular) to see the Kanban board click on two circles icon on page control
- https://reidgould.github.io/tiddlywiki-dropboard/ (proof of concept)

**Reference**
-  https://www.projectmanager.com/kanban
-  https://en.wikipedia.org/wiki/Kanban_(development

**Comment by NicolasPetton** [comment]
A kanban board would be nice indeed, but it's not super high in my todo-list to be honest, and it would be quite a bit of work to implement.
